yubikey-luks-suspend

```diff
@@ -48,7 +48,7 @@ mount_barrier() {
 umount_bind() {
     local p
     for p in ${BIND_PATHS}; do
-        mountpoint -q "${INITRAMFS_DIR}${p}" && umount "${INITRAMFS_DIR}${p}"
+        mountpoint -q "${INITRAMFS_DIR}${p}" && umount -l "${INITRAMFS_DIR}${p}"
```

Comment:
Why would the initramfs hand during unmount?
In which other cases than network file systems s this necessary?

Reply:
Multiple users reported that `umount` without `-l` flag doesn't always work, see https://github.com/cornelinux/yubikey-luks/issues/68

I guess there is some race condition when something is still accessing mounted paths when `umount` is called and without `-l` option it only tries it once and then bails out if it didn't work even when retrying it just after will work just fine which is what `-l` option does.

Reply:
You may note we use [umount -l](https://github.com/cornelinux/yubikey-luks/blob/master/yubikey-luks-suspend#L56) few lines below already with a comment why so this one fixes omission.